mobile/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,64 @@
+import CommunitySlider from '@react-native-community/slider';
+import React, { useState } from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { convertInputToDate } from '~/bundles/common/helpers/helpers';
+import { useCallback, useWindowDimensions } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type SliderProperties = {
+    style?: StyleProp<ViewStyle>;
+};
+
+const minSliderValue = 0;
+const maxSliderValue = 120;
+const valueWidth = 70;
+
+const Slider: React.FC<SliderProperties> = ({ style: viewStyle }) => {
+    const [value, setValue] = useState(minSliderValue);
+    const { width } = useWindowDimensions();
+    const monthTitle = convertInputToDate(value);
+    const resetValue = useCallback(() => {
+        setValue(minSliderValue);
+    }, []);
+    const offset = (valueWidth / maxSliderValue) * value;
+    const leftValue = (value * width) / maxSliderValue - offset;
+    return (
+        <View style={[globalStyles.p5, viewStyle]}>
+            <View style={[globalStyles.flexDirectionRow]}>
+                <Text category={TextCategory.LABEL} style={styles.label}>
+                    Experience
+                </Text>
+                <Text category={TextCategory.LABEL} style={styles.label_icon}>
+                    *
+                </Text>
+            </View>
```

Comment:
Firstly, we have a FormField component already with this logic. Secondly, this component should only represent data passed via props, so that we can use it in multiple places. So for now you don't need any date functions and so on, just representing provided data

Reply:
So there should be basic slider props, and some values under the slider, like"no", which also should be passed via props

Reply:
fix